### PR TITLE
Dev Private Setting

### PR DESCRIPTION
**Pull request by jamil-source**
GLHF

### CODE REVIEW

File: frontend/src/components/login/index.jsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ const Login = ({getToken}) => {
     })
   };
 
-  if(redirect) return <Redirect to="/private-settings"/>;
+  if(redirect) return <Redirect to="/change-password"/>;
```
Comment (stenwall) [inline frontend/src/components/login/index.jsx:33]: Why redirect to this page after login? Shouldn't we be redirected to the preference-page?

**Review by stenwall — APPROVED** [review]
LGTM! 🎉

File: frontend/src/components/login/index.jsx (hunk 1 above)
Comment (jamil-source) [reply frontend/src/components/login/index.jsx:33]: Temporary, this Will be changed when other components such as menu and so on is built

**Comment by jamil-source** [comment]
this will close #53

**Pull request merged**